sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -2,62 +2,62 @@
 
 ## Introduction
 
-Azure SDK clients provide an interface to Azure services by translating library calls to REST requests. 
+Azure SDK clients provide an interface to Azure services by translating library calls to REST requests.
 
 The schematized body in the request or response, known as the message body, can be exposed in the Azure SDK client one of two ways:
 
 Most Azure SDK Clients expose methods that take ['model types'](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-model-types) as parameters, C# classes which map to the message body of the REST call. Those methods can be called here '**standard model methods**',
 
 However, some clients expose methods that mirror the message body directly. Those methods are called here '**protocol methods**', as they provide more direct access to the REST protocol used by the client library.
 
-### Pet's Example 
+### Pet's Example
 
-To compare the two approaches, imagine a service that stores information about pets, with a GetDog and SetDog pair of operations. 
+To compare the two approaches, imagine a service that stores information about pets, with a GetDog and SetDog pair of operations.
 
 Pets are represented in the message body as a JSON object:
 
 ```json
 {
-        "name": "Buddy",
-        "id": 2,
-        "color": "Brown"
+    "name": "Buddy",
+    "id": 2,
+    "color": "Brown"
 }
 ```
 
 An API using model types could be:
 
 ```csharp
-        // This is an example model class
-        public class Pet
-        {
-                string Name { get; }
-                int Id { get; }
-                string Color { get; }
-        }
-
-        Response<Pet> GetDog(string dogName);
-        Response SetDog(Pet dog);
+// This is an example model class
+public class Pet
+{
+        string Name { get; }
+        int Id { get; }
+        string Color { get; }
+}
+
+Response<Pet> GetDog(string dogName);
+Response SetDog(Pet dog);
 ```
 
 While the protocol methods version would be:
 
 ```csharp
-        // Request: {
-        //      "name": "Buddy",
-        // }
-        // Response: {
-        //      "name": "Buddy",
-        //      "id": 2,
-        //      "color": "Brown"
-        // }
-        Response GetDog(RequestContent requestBody, RequestOptions options = default);
-        // Request: {
-        //      "name": "Buddy",
-        //      "id": 2,
-        //      "color": "Brown"
-        // }
-        // Response: {}
-        Response SetDog(RequestContent requestBody, RequestOptions options = default);
+// Request: {
```

Comment:
I'm a little unsure what we are trying to convey with these commented out JSON objects.

Reply:
I guess the original intention is to provide examples of request/response contents.

Reply:
@JoshLove-msft, good point.  @archerzz is correct.  In DPG clients, the JSON schema for inputs and outputs is illustrated in doc comments for users to help them know what to put in RequestContent and find in Response.